src/types.ts

```diff
@@ -77,6 +77,29 @@ export enum GameOption {
     MULTIGUESS = "Multiguess",
 }
 
+export enum GameOptionCommand {
```

Comment:
If you convert this to a dictionary that maps from `GameOption` to a string, then you don't need to use an index when you're mapping over it in `discord_utils`. You could do something like `gameCommandArray[option]` similar to how you did `optionStrings[option]`

Reply:
okay